variables.tf

```diff
@@ -130,3 +130,31 @@ variable "require_http_tokens" {
   description = "Whether to enable IMDSv2 on the Tamr EC2 Instance"
   default     = true
 }
+
+#
+# Startup Script
+#
+variable "tamr_config_file" {
+  type        = string
+  description = "Override generated tamr configuration. The tamr configuration is specified using a yaml file, in the format that is documented (https://docs.tamr.com/previous/docs/configuration-configuring-unify#section-setting-configuration-variables) for configuring “many variables” at once."
+}
+
+variable "tamr_zip_uri" {
+  type        = string
+  description = "gcs location to download tamr zip from"
+}
+
+variable "tamr_instance_install_directory" {
+  # Get it?, DataTamer :p
+  default     = "/data/tamr"
+  type        = string
+  description = "directory to install tamr into"
+}
+
+#
+# file system
+#
+variable "tamr_filesystem_bucket" {
```

Comment:
Is this only meant to support scale out? This only needs to be set for scale out instances.

Reply:
My understanding was this module is only meant for scale out (I just mimicked the same module for gcp), but if that's not the case I can refactor.